bin/dockerize.sh

```diff
@@ -3,10 +3,10 @@
 # Define the directories
 directories=(
     "osc-dm-product-srv"
-    "osc-dm-search-srv"
-    "osc-dm-proxy-srv"
-    "osc-dm-monitor-srv"
-    "osc-dm-registrar-srv"
+#    "osc-dm-search-srv"
+#    "osc-dm-proxy-srv"
+#    "osc-dm-monitor-srv"
+#    "osc-dm-registrar-srv"
```

Comment:
Is this intentional? I would think that we want all of these to be enabled, so that you can easily rebuild the entire env

Reply:
Left in by accident. uncommented all repos so dockerize should rebuild entire environment